src/css/profile/wearable/changeable/theme-circle/theme.changeable.less

```diff
@@ -260,8 +260,9 @@
 @bg_scroller_bouncing_effect_bottom_edge: "./images/Scroller/bouncing_circle_bottom_edge.png";
 @bg_scroller_bouncing_effect_left_glow: "./images/Scroller/bouncing_circle_left_glow.png";
 @bg_scroller_bouncing_effect_left_edge: "./images/Scroller/bouncing_circle_left_edge.png";
-@color_scroller_bouncing_effect_glow: B018E;
-@color_scroller_bouncing_effect_edge: B019;
+@color_scroller_bouncing_effect: #4297ff;
```

Comment:
this color is defined in `theme.colorPalette.less` as common `@controller-on-color`

Reply:
Thanks. Done.